geps/gep-1713.md

```diff
@@ -0,0 +1,220 @@
+  // this Gateway a child of another.
+  AttachTo LocalObjectReference `json:"attachTo"`
+}
+```
+
+#### YAML
+```yaml
+apiVersion: gateway.networking.k8s.io/v1beta1
+kind: Gateway
+metadata:
+  name: child-gateway
+spec:
+  gatewayClassName: example
+  infrastructure:
+    attachTo:
+      name: parent-gateway
+      kind: Gateway
+      group: gateway.networking.k8s.io
+  listeners:
+  - name: metrics
+    port: 8080
+    protocol: HTTP
+```
+
+### Semantics
+
+#### Gateway Attaching
+
+Gateways that _do not_ reference another Gateway (using `spec.infrastructure.attachTo`) are considered "primary" Gateways. A "child" Gateway is one where `spec.infrastructure.attachTo` field is set to a "primary" Gateway - making the "primary" Gateway a "parent". A "sibling" is a Gateway that shares a "parent" with another "child" Gateway. A "child" Gateway can only attach to Gateways in the same namespace.
+
+Failure to attach a "child" Gateway to another MUST result in the "child" Gateway setting their `Accepted` condition to `False`.
+
+This MUST happen in the following scenarios:
+- Attaching to a non-existing Gateway
+
+If a Gateway's `Accepted` condition is `False` then the conditions 'Programmed' and 'Ready' (if supported) MUST be set to `False`.
+
+Invalid Gateways MUST be rejected by the implementation during creation or update. This MUST happen in the following scenarios:
+- Attaching to another "child" Gateway
+- Attaching to oneself (self-referential)
+- Attaching to a Gateway with a different `GatewayClassName`
+
+#### Route Attaching
+
+Routes MUST be able to specify a "child" Gateways as a `parentRef` and make use of the fields in `ParentReference` to help target a specific listener.
+
+
+```yaml
+apiVersion: gateway.networking.k8s.io/v1beta1
+kind: HTTPRoute
+metadata:
+  name: httproute-example
+spec:
+  parentRefs:
+  - name: child-gateway
+    sectionName: metrics
+```
+
+#### Merging Spec Fields
+
+##### GatewayClassName
+
+The `GatewayClassName` MUST be set on "child" and "parent" Gateways and be equivalent.
+
+##### Listeners
+
+Implementations MUST treat the "parent" Gateway as having the concatenated list of all listeners from itself and "child" Gateways.
+
+Validation of this list of listeners is subject to the discussion here: https://github.com/kubernetes-sigs/gateway-api/issues/1842
+
+eg.
+```yaml
+apiVersion: gateway.networking.k8s.io/v1beta1
+kind: Gateway
+metadata:
+  name: parent-gateway
+spec:
+  gatewayClassName: example
+  listeners:
+  - name: HTTP
+    port: 80
+    protocol: HTTP 
+---
+apiVersion: gateway.networking.k8s.io/v1beta1
+kind: Gateway
+metadata:
+  name: child-gateway
+spec:
+  gatewayClassName: example
+  infrastructure:
+    attachTo:
+      name: parent-gateway
+      kind: Gateway
+      group: gateway.networking.k8s.io
+  listeners:
+  - name: metrics
+    port: 8080
+    protocol: HTTP    
+```
+
+With this configuration, the realized "primary" Gateway should listen on port `80` and `8080`.
```

Comment:
It feels unusual that the child would be able to cause the parent to listen on additional ports since this would make it relatively difficult for a user to understand the entire set of ports the Gateway is listening on.  Is merging new ports from the child gateway filling a specific requirement?

Reply:
I thought the introduction to the GEP covers the _why_. After reading it do you have further questions or suggestions to reword it?

Reply:
hi @briantkennedy just following up - as it's been a while